genologics/descriptors.py

```diff
@@ -238,8 +238,10 @@ def __getitem__(self, key):
 
     def __setitem__(self, key, value):
         self._lookup[key] = value
```

Comment:
I'm looking a bit further into the changes in this function! I can discuss it on Monday.

Reply:
OK, now I've gone made a new PR that's WIP and I will send in after this is merged.